### PR TITLE
Update Web-Interface.md

### PR DESCRIPTION
**Pull request by Petrprogs**
Add new review and upload new png to resources dir

### CODE REVIEW

**Review by ckerr — APPROVED** [review]
Thank you again! :green_heart:

**Comment by Petrprogs** [comment]
> Thank you again! green_heart

Thank you for creating such a wonderful project :)

File: docs/Web-Interface.md [hunk 1]
```diff
@@ -98,4 +98,28 @@ If the web browser and the Transmission daemon are on the machine you can use ht
 | 6                 | Progress bar showing the current progress in downloading the torrent            |
 | 7                 | Pause selected torrent                                                          |
 
+### Torrent inspector ###
+
+
+| Number On Picture | Description                                                                                                                                |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| 1                 | Name of torrent                                                                                                                            | 
+| 2                 | In current case: Transmission have 3.65 GB of file data from the total volume of downloaded files, which is 100%                           |
```
Comment (RobFog) [inline docs/Web-Interface.md:107]: This should probably read „Transmission has“. The app is singular.

Comment (Petrprogs) [reply docs/Web-Interface.md:107]: Ok, i fix it in next PR!

**Pull request merged**